decentraland/components/presentational/AccountCreationProgress.js

```diff
@@ -43,15 +43,13 @@ export default function AccountCreationProgress(props) {
     default:
   }
 
+  if (!waitingForAccountSetup) return null;
```

Comment:
You could have kept it as a ternary operator if you wanted, although I do like this new approach even better

Reply:
Resolving this